README.adoc

```diff
@@ -133,10 +141,10 @@ This endpoint has been provided for you to observe the results of an unhealthy p
 
 == Configuring readiness and liveness probes
 
-You will configure {kube} readiness and liveness probes. Readiness probes are responsible for 
-determining that your application is ready to accept requests. If it's not ready, traffic 
-won't be routed to the container. Liveness probes are responsible for determining when a 
-container needs to be restarted. 
+You will configure {kube} readiness and liveness probes.
+Readiness probes are responsible for determining that your application is ready to accept requests.
+If it's not ready, traffic won't be routed to the container.
+Liveness probes are responsible for determining when a container needs to be restarted. 
```

Comment:
Readiness probes are responsible for determining that your application is ready to accept requests.
---->
Readiness probes determine whether your application is ready to accept requests.


Liveness probes are responsible for determining when a container needs to be restarted. 
--->
Liveness probes determine whether a container needs to be restarted.